target/linux/mediatek/filogic/base-files/lib/preinit/10_fix_eth_mac.sh

```diff
@@ -10,6 +10,11 @@ preinit_set_mac_address() {
 		ip link set dev game address "$(cat $key_path/LANMAC)"
 		ip link set dev eth1 address "$(cat $key_path/WANMAC)"
 		;;
+	asus,rt-ax59u)
+		CI_UBIPART="UBI_DEV"
+		addr=$(mtd_get_mac_binary_ubi "Factory" 0x4)
+		ip link set dev eth0 address "$addr"
+		;;
```

Comment:
We still have random mac on `eth0`. This don't work for the Mediatek switch. I noticed this on several devices. So we can drop this block because it's useless. We set correct MAC in `02_network` script.

P.S. `eth0` string in `asus,tuf-*` block is also useless.